coremltools/converters/mil/frontend/torch/test/test_executorch_e2e.py

```diff
@@ -92,6 +94,139 @@ def forward(self, a, x, b):
             backend=("mlprogram", "fp16"),
         )
 
+    def test_softmax(self):
+        class LinearModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.softmax = torch.nn.Softmax()
+
+            def forward(self, x):
+                z = self.softmax(x)
```

Comment:
nit: why not just `return self.softmax`?

Reply:
This is to be consistent with [ExecuTorch](https://github.com/pytorch/executorch/blob/main/examples/models/toy_model/model.py#L83)

Reply:
Not sure why is those two cases are different,
just curious

Reply:
The same yeah... I'll just simplify it to `return self.softmax(x)` then